src/status_im/chat/models/delete_message_for_me.cljs

```diff
@@ -0,0 +1,69 @@
+(ns status-im.chat.models.delete-message-for-me
+  (:require [re-frame.core :as re-frame]
+            [status-im.ethereum.json-rpc :as json-rpc]
+            [status-im.utils.fx :as fx]
+            [taoensso.timbre :as log]))
+
+(defn- update-db-clear-delete-message-for-me-undo-timer
+  [db chat-id message-id]
+  (when (get-in db [:messages chat-id message-id])
+    (update-in db
+               [:messages chat-id message-id]
+               dissoc
+               :deleted-for-me-undoable-till)))
+
+(defn- update-db-delete-message-for-me-locally
+  [db chat-id message-id undo-time-limit-ms]
+  (when (get-in db [:messages chat-id message-id])
+    (update-in db
+               [:messages chat-id message-id]
+               assoc
+               :deleted-for-me? true
+               :deleted-for-me-undoable-till (+ (js/Date.now)
```

Comment:
We have a utility namespace for date & time operations. I think you can replace this call and the other one in `update-db-undo-delete-message-for-me-locally` with the function `status-im.utils.datetime/timestamp`.

Reply:
done